.github/workflows/nx.yml

```
@@ -17,13 +17,37 @@
+        name: Checkout [Pull Request]
+        if: ${{ github.event_name == 'pull_request' }}
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+      - uses: actions/checkout@v4
+        name: Checkout [Default Branch]
+        if: ${{ github.event_name != 'pull_request' }}
+        with:
+          fetch-depth: 0
+      - name: "Setup"
+        uses: ./.github/actions/setup
```

actionlint (1.7.4)

description is required in metadata of "Setup" action at "/home/jailuser/git/.github/actions/setup/action.yml"

(action)